### PR TITLE
Adding DEV SNAPSHOT from 2016-12-01 for Ubuntu 15.10

### PR DESCRIPTION
**Pull request by rbukovansky**

### CODE REVIEW

File: versions/DEVELOPMENT-SNAPSHOT-2016-12-01-a.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 binaries:
   osx: https://swift.org/builds/development/xcode/swift-DEVELOPMENT-SNAPSHOT-2016-12-01-a/swift-DEVELOPMENT-SNAPSHOT-2016-12-01-a-osx.pkg
+  ubuntu15.10: https://swift.org/builds/development/ubuntu1510/swift-DEVELOPMENT-SNAPSHOT-2016-11-01-a/swift-DEVELOPMENT-SNAPSHOT-2016-11-01-a-ubuntu15.10.tar.gz
```
Comment (ikesyo) [inline versions/DEVELOPMENT-SNAPSHOT-2016-12-01-a.yaml]: Why `2016-11-01`?

Comment (rbukovansky) [inline versions/DEVELOPMENT-SNAPSHOT-2016-12-01-a.yaml]: Oops! Sorry! #MondayMorning

**Comment by rbukovansky** [comment]
Closing. Bad idea.

**Comment by kylef** [comment]
Btw, you can use the `extract.py` script to add new version files instead of manually doing it. Check out https://github.com/kylef/swiftenv-api#development-environment

This should prevent any user-error while creating the versions.

**Comment by rbukovansky** [comment]
@kylef OK. Next time. Thanks.

**Pull request closed without merging**